tests/unit-tests/test-class-quiz.php

```diff
@@ -2096,4 +2096,356 @@ public function testQuizFooterActions_WhenInProgressButInLearningMode_DoesNotRen
 		/* Assert */
 		$this->assertStringNotContainsString( 'Pending teacher grade', $result );
 	}
+
+	public function testQuizFooterActions_WhenPassedInLearningMode_ShowsTheNextLessonButton() {
```

Comment:
Is there a method named `quiz_footer_actions` that this test case covers? If not, I think it may be [named](https://senseip2.wordpress.com/2022/03/17/unit-tests-conventions/#naming-convention-for-tests) incorrectly.

Reply:
I named it this way because basically I'm checking the Quiz Actions block in the Quiz footer here, so wasn't sure how to name it according to `testMethodName_Conditions_Expectation` convention as it requires it to be a method. Now I've updated it here https://github.com/Automattic/sensei/pull/7202/commits/43b4867e1ef51dc40f8382337e061925ed265a0d to match the method name https://github.com/Automattic/sensei/blob/bcd43ea75c074a0ae8b45c8376dd795683514ee3/includes/class-sensei-quiz.php#L1850 which is actually rendering the action buttons.